src/aws/osml/model_runner/sink/kinesis_sink.py

```diff
@@ -2,11 +2,12 @@
 
```

Comment:
I generally note - can we wrap some try catches around the logic in the submethods and have it through a custom Kinesis related exception? Otherwise LGTM